example/src/__snapshots__/App.test.jsx.snap

```diff
@@ -57,5 +57,16 @@ exports[`Basic test should render 1`] = `
   <p>
     Photo by Louis Hansel @shotsoflouis on Unsplash
   </p>
+  <p>
+    Test variable: 
```

Comment:
Should `TEST_VARIABLE` also be added to a .env.test file so that the environment variable gets picked up in the snapshot?

Reply:
Doh!  Good idea.